index.js

```diff
@@ -192,7 +203,7 @@ Iso8583.prototype.decode = function(buffer, $meta, context, log) {
             } else {
                 $meta.opcode = String(message[3] || '').substr(0, 2);
             }
-            $meta.trace = `${(message.mtid || '00').substr(0, 2)}${message[11]}`;
+            $meta.trace = this.getTrace(message);

```

Comment:
I suggest using ut-function.template (https://github.com/softwaregroup-bg/ut-function/tree/master/packages/template)
```js
$meta.trace = this.template(message)
```
this.template can be set to:
```js
const template = require('ut-function.template');
this.template = template(config.traceFields || "${(message.mtid || '00').substr(0, 2)}${message[11]}", ['message']);
```